src/Client/Mailer.php

```diff
@@ -17,6 +17,9 @@ public function __construct(
     ) {
     }
 
+    /**
+     * @psalm-param array{templateKey: string, ...} $requestBody
```

Comment:
@bdsl This is nice. Do you think there's also some value in mentioning it should also have `recipientEmailAddress` and `params` too?

Reply:
@alihejazi Done in 3cd1a96947ab . Because I did that Psalm forced me to add a null check for the person's email address in `CreatePasswordResetToken.php`.